development/cpp/configuring_an_ide/visual_studio_code.rst

```diff
@@ -10,7 +10,8 @@ Importing the project
 ---------------------
 
 - Make sure the C/C++ extension is installed. You can find instructions in
-  the `official documentation <https://code.visualstudio.com/docs/languages/cpp>`_.
+  the `official documentation <https://code.visualstudio.com/docs/languages/cpp>`_. Alternatively, `clangd <https://open-vsx.org/extension/llvm-vs-code-extensions/vscode-clangd>`_ can be used instead.
+- When using the clangd extension, run ``scons compiledb=True``.
```

Comment:
```suggestion
- When using the clangd extension, run ``scons compiledb=yes``.
```

Reply:
Ah. This makes sense (it's more in line with the way scons is used all over the documentation). I'm not very familiar with scons and after reading SConstruct at projects root I was under impression it was expecting ``True``. compiledb is initialized to ``False`` there. ``yes`` certainly works better, though.